docs/docs/APIdocs/layers/nn/Loss Functions/HingeEmbeddingCriterion.md

```diff
@@ -0,0 +1,81 @@
+## HingeEmbeddingCriterion ##
+
+
+**Scala:**
+``` scala
+criterion = HingeEmbeddingCriterion(margin: Double = 1, sizeAverage: Boolean =true)
```

Comment:
this code will throw error when user copy/paste it on the scala REPL. Maybe we should make it this way? 
```scala
val criterion = HingeEmbeddingCriterion(margin = 1, sizeAverage =true)
```
This way it omitts the actual type of each argument. - Yet I think if user needs to know the actual type of the parameter, they can refer to ScalaDoc.